packages/core/src/State.ts

```diff
@@ -182,7 +188,7 @@ export class State<
     this._sessionid = config._sessionid;
     this.event = this._event.data;
     this.historyValue = config.historyValue || {};
-    this.actions = config.actions || [];
+    this.actions = [...(config.actions || [])];
```

Comment:
why do we need to copy `config.actions` here?

Reply:
591718c29abe83714f87b97018fc4de0e201b1d7